test/fail_compilation/constraints_func3.d

```diff
@@ -2,43 +2,43 @@
 EXTRA_FILES: imports/constraints.d
 TEST_OUTPUT:
```

Comment:
Perhaps make a copy of this file -> `constraints_func4.d` and add:
```suggestion
REQUIRED_ARGS: -verrors=context
TEST_OUTPUT:
```
So there's a good example in the testsuite of how this interacts with the `-verrors=context` switch.